src/Interfaces/IChainflip.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+/// @title Interface for Chainflip Vault contract
+/// @custom:version 1.0.0
+interface IChainflipVault {
+    /// @notice Swaps native token to any supported asset on any supported chain
+    /// @param dstChain Destination chain for the swap
+    /// @param dstAddress Address where the swapped tokens will be sent to on the destination chain
+    /// @param dstToken Token to be received on the destination chain
```

Comment:
would suggest to update this to make it clear that this is a chainflip-internal ID, not a token address